docs/schema-v2.md

```diff
@@ -5,12 +5,413 @@ This specification defines a format for a set of rules for the purpose of custom
+| `rotation`        | Rotation object | Specifies the rotation angles of the skybox.                                                                             |         :x:        | [0,0,0] for static/axis, 1 for rotationSpeed |
+
+**Example**
+```json
+{
+  "fade": {
+    "startFadeIn": 1000,
+    "endFadeIn": 2000,
+    "startFadeOut": 3000,
+    "endFadeOut": 4000
+  },
+  "maxAlpha": 0.5,
+  "transitionSpeed": 0.8,
+  "sunSkyTint": false,
+  "changeFog": true,
+  "fogColors": {
+    "red": 0.2,
+    "green": 0.9,
+    "blue": 0.6,
+    "alpha": 1.0
+  },
+  "shouldRotate": true,
+  "rotation": {
+    "static": [
+      216,
+      288,
+      144
+    ],
+    "axis": [
+      36,
+      108,
+      72
+    ]
+  }
+}
+```
+
+### Conditions Object
+
+Specifies when and where a skybox should render. All fields are optional.
+
+**Specification**
+
+|    Name   |         Datatype        |                                  Description                                  |             Default value            |
+|:---------:|:-----------------------:|:-----------------------------------------------------------------------------:|:------------------------------------:|
+| `biomes`  | Array of Namespaced Ids | Specifies a list of biomes that the skybox should be rendered in              |       Empty Array (all biomes)       |
+| `worlds`  | Array of Namespaced Ids | Specifies a list of worlds that the skybox should be rendered in              |       Empty Array (all worlds)       |
+| `weather` | Array of Weathers       | Specifies a list of weather conditions that the skybox should be rendered in  | Empty Array (all weather conditions) |
+| `xRanges` | Array of MinMax Entries | Specifies a list of coordinates that the skybox should be rendered between    |   Empty Array (all x coordinates)    |
+| `yRanges` | Array of MinMax Entries | Specifies a list of coordinates that the skybox should be rendered between    |   Empty Array (all y coordinates)    |
+| `zRanges` | Array of MinMax Entries | Specifies a list of coordinates that the skybox should be rendered between    |   Empty Array (all z coordinates)    |
+
+
+**Example**
+```json
+{
+  "biomes": [
+    "minecraft:desert",
+    "minecraft:desert_hills",
+    "minecraft:desert_lakes"
+  ],
+  "worlds": [
+    "minecraft:overworld"
+  ],
+  "weather": [
+    "rain",
+    "thunder"
+  ],
+  "xRanges": [
+    {
+      "min": -100.0,
+      "max": 100.0
+    }
+  ],
+  "yRanges": [
+    {
+      "min": 50.0,
+      "max": 60.0
+    },
+    {
+      "min": 100.0,
+      "max": 110.0
+    }
+  ],
+  "zRanges": [
+    {
+      "min": -100.0,
+      "max": 100.0
+    }
+  ]
+}
+```
+
+### Decorations Object
+Stores all specifications for sun and moon configuration. For optimum results, the moon texture should be a 4 wide, 2 high stacked texture.
```

Comment:
Possibly replace the ending with "the moon texture should mimic the vanilla moon texture"?  That way, if Mojang ever updates it (for some cursed reason) the docs are still correct.